.github/workflows/links.yml

```diff
@@ -0,0 +1,57 @@
+name: Links
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 15 * * 1'  # Run every Monday at 3:00 PM UTC
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository  
+        uses: actions/checkout@v4
+
+      - name: Submodule
+        run: git submodule update --init --recursive
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Serve App Locally
+        run: yarn run dev &
+
+      - name: Wait for App to Start
+        run: sleep 20
+      
+      # This will restore the lychee cache
+      - name: Restore lychee cache
+        uses: actions/cache@v4
+        with:
+          path: .lycheecache
+          key: cache-lychee-${{ github.sha }}
+          restore-keys: cache-lychee-
+
+      # This will run the link checker on all markdown files in the pages directory      
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1
+        with:
+          args: --base http://localhost:3000  --verbose --no-progress --accept 200,204,429,403  './pages/**/*.md' --cache --max-cache-age 1d  http://localhost:3000 
+          token: ${{secrets.AUTH_TOKEN}}
+        
+      # This will create an issue with the report if the link checker fails      
+      - name: Create Issue From File
```

Comment:
If we don't close the issue of the previous iteration we can have duplicated issues for the same purpose "Link Checker Report". I suggest in case of error we do something like:
- If there is not "Link Checker Report" we create the new issue.
- If there is a "Link Checker Report" issue opened, we can add the results as a new comment to that issue.

Reply:
Thanks for the suggestions, Making this changes.
I think we need to use the GitHub API to check whether an issue has already been created or not.

Reply:
Yes. We are doing that kind of logic in the workflows in the Community repo to manage the Open Community Working Meetings creation/agenda.